### PR TITLE
Update OE.md

### PR DESCRIPTION
**Pull request by BernardPuel**
Hello Robert,
Here is my proposition to render this article more generic and more fit to the principle of OE.
Take what you feel acceptable for other partners (you are the boss) !

I do think these articles below may also be more generic if you remove what is not completely linked to OE (but more in a dedicated article like the getting started):
# Bootloaders and eMMC partitions
# Flashing build artifacts
# Proprietary firmware blob

Documentation structure is not easy to manage with the diversity but more you describe generic thinks more you simplify your work ....  Cheers !

### CODE REVIEW

File: Reference-Platform/CECommon/OE.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## OpenEmbedded and Yocto
 
-This page provides instructions to get started with OpenEmbedded and the Yocto Project on the DragonBoard 410c and HiKey. 
+This page provides instructions to get started with OpenEmbedded and the Yocto Project. It aims at being generic for any board support.
+The board diversity should be addressed through dedicated BSP layer then MACHINE choice.
```
Comment (ndechesne) [inline Reference-Platform/CECommon/OE.md:4]: I agree that this page needs some improvements. It started as a copy/paste of my initial OE setup page for QCOM boards. However I don't think we want that page to be a reference page for 'any board'. This is the Linaro OE RPB getting started page, so the focus is on the BSP that *we* support, not any. So I think the intro should reflect that, and maybe explain what the OE RPB is.

File: Reference-Platform/CECommon/OE.md [hunk 2]
```diff
@@ -11,15 +12,12 @@ This wiki is not an introduction on OpenEmbedded or Yocto Project. If you are no
 
 In this wiki, we assume that the reader is familiar with basic concepts of OpenEmbedded.
 
-The support for DragonBoard 410c is available in the [meta-qcom BSP layer](http://git.yoctoproject.org/cgit/cgit.cgi/meta-qcom).
-
-The support for HiKey is available in the [meta-96boards BSP layer](https://github.com/96boards/meta-96boards).
-
-These layers have been tested with OpenEmbedded Core layer, and are expected to work with any other standard layers and of course any OpenEmbedded based distributions.
+The support for any board is available in the dedicated BSP Layers. These layers have been tested with OpenEmbedded Core layer, and are expected to work with any other standard layers and of course any OpenEmbedded based distributions.
```
Comment (ndechesne) [inline Reference-Platform/CECommon/OE.md:15]: I think it is fine to list all boards that we support (or intend to) here.

**Comment by koenkooi** [comment]
looks good to me

**Comment by ndechesne** [comment]
not to me.  i am composing a review,.

**Review by ndechesne — COMMENTED** [review]
i would have preferred if this change was not merged.. anyways, can you please consider my review and provide feedback?

File: Reference-Platform/CECommon/OE.md (hunk 2 above)
Comment (BernardPuel) [reply Reference-Platform/CECommon/OE.md:15]: The sense of my update here was to say: when the reader arrives in this page, he already knows that his board is supported by Linaro OE RPB distribution. The same for board support (any means any of the linaro board supported).

Comment (ndechesne) [reply Reference-Platform/CECommon/OE.md:15]: the sentence that really triggered my review is "It aims at being generic for any board support." . I don't want to say something like this. We can have a less generic introduction i believe.

**Pull request merged**